app/components/Feedback/index.tsx

```diff
@@ -59,7 +59,7 @@ const Feedback = ({
       <CompositeButton secondary>
         <Action
           pageid={pageid}
-          showText={!!labels}
+          showtext={(!!labels).toString()}
```

Comment:
I get `showText` vs `showtext` might have been a problem,, but what's with the `.toString()` part?

Reply:
`showtext` expects a string, so `false` was causing errors that `"false"` fixes...

Reply:
:suspect: